databases/week1/queries-hw1.sql

```diff
@@ -0,0 +1,84 @@
+/*Find out how many tasks are in the task table*/
+SELECT
+  COUNT(*)
+FROM
+  task;
+  /*Find out how many tasks in the task table do not have a valid due date*/
+SELECT
+  COUNT(*)
+FROM
+  task
+WHERE
+  due_date is NULL;
+  /*Find all the tasks that are marked as done*/
+SELECT
+  *
+FROM
+  task
+WHERE
+  due_date is NULL;
+  /*Find all the tasks that are not marked as done*/
+SELECT
+  *
+FROM
+  task
+  JOIN status ON task.status_id = status.id
+WHERE
+  status.name = "Done";
```

Comment:
This one doesn't match the description either, it looks like there was some mismatch between the exercises 😁